server/src/test/integration/org/apache/cassandra/sidecar/routes/StreamStatsIntegrationTest.java

```diff
@@ -42,109 +40,133 @@
-    void streamStatsTest(VertxTestContext context, ConfigurableCassandraTestContext cassandraTestContext) throws Exception
+    @CassandraIntegrationTest(numDataDirsPerInstance = 4, nodesPerDc = 2, network = true)
+    void streamStatsTest(CassandraTestContext cassandraTestContext)
     {
-        BBHelperDecommissioningNode.reset();
-        UpgradeableCluster cluster = cassandraTestContext.configureAndStartCluster(
-        builder -> builder.withInstanceInitializer(BBHelperDecommissioningNode::install));
-        IUpgradeableInstance node = cluster.get(2);
-
-        createTestKeyspace();
-        createTestTableAndPopulate();
-
-        startAsync("Decommission node" + node.config().num(),
-                   () -> node.nodetoolResult("decommission", "--force").asserts().success());
-        AtomicBoolean hasStats = new AtomicBoolean(false);
-        AtomicBoolean dataReceived = new AtomicBoolean(false);
-
-        // Wait until nodes have reached expected state
-        awaitLatchOrThrow(BBHelperDecommissioningNode.transientStateStart, 2, TimeUnit.MINUTES, "transientStateStart");
-
-        // optimal no. of attempts to poll for stats to capture streaming stats during node decommissioning
-        loopAssert(15, 200, () -> {
-            StreamsProgressStats progressStats = streamStats(hasStats, dataReceived);
-            assertThat(hasStats).isTrue();
-            assertThat(dataReceived)
-            .describedAs("Stream Progress Stats - totalFilesReceived:" + progressStats.totalFilesReceived() +
-                         " totalBytesReceived:" + progressStats.totalBytesReceived())
-            .isTrue();
-        });
-        ClusterUtils.awaitGossipStatus(node, node, "LEFT");
-        BBHelperDecommissioningNode.transientStateEnd.countDown();
+//        BBHelperDecommissioningNode.reset();
+        UpgradeableCluster cluster = cassandraTestContext.cluster();
+//        cassandraTestContext.configureAndStartCluster(
+//        builder -> builder.withInstanceInitializer(BBHelperDecommissioningNode::install));
 
-        context.completeNow();
-        context.awaitCompletion(2, TimeUnit.MINUTES);
+        createTestKeyspace(Map.of("datacenter1", 2));
+        QualifiedTableName tableName = createTestTable(
+        "CREATE TABLE %s ( \n" +
+        "  race_year int, \n" +
+        "  race_name text, \n" +
+        "  cyclist_name text, \n" +
+        "  rank int, \n" +
+        "  PRIMARY KEY ((race_year, race_name), rank) \n" +
+        ");");
+        // craft inconsistency for repair
+        populateDataAtNode2Only(cluster, tableName);
+
+        // Poll stream stats while repair is running in the background.
+        CountDownLatch testStart = new CountDownLatch(1);
+        IUpgradeableInstance node = cluster.get(1);
+        AtomicReference<Throwable> nodetoolError = new AtomicReference<>();
+        startAsync("Repairing node" + node.config().num(),
```

Comment:
Changed the stream trigger from decommission to repair. The test also does not need bytebuddy to intercept, hence removing it. 
We should avoid bytebuddy at the best, to avoid dependency on Cassandra's implementation details.